static/src/js/components/TemporalDisplay/TemporalSelectionDropdownToggle.jsx

```diff
@@ -9,6 +9,7 @@ const TemporalSelectionDropdownToggle = ({ onToggleClick }) => (
   <Dropdown.Toggle
     variant="inline-block"
     id="temporal-selection-dropdown"
+    aria-label="temporal-selection-dropdown"
```

Comment:
This is not a good aria-label. This should be some human readable text. It is meant to add context to users with assistive technology. Something like "Open temporal filters" might work well here.